src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/LicenceRepository.kt

```diff
@@ -22,8 +22,10 @@ interface LicenceRepository : JpaRepository<Licence, Long>, JpaSpecificationExec
         JOIN l.submittedBy com
         WHERE (l.actualReleaseDate = CURRENT_DATE OR l.conditionalReleaseDate = CURRENT_DATE) 
         AND l.statusCode = 'SUBMITTED'
-        AND EXISTS 
-                (SELECT 1 FROM AuditEvent ae WHERE l.id = ae.licenceId AND ae.detail LIKE '%APPROVED%')
+        AND (
+            (EXISTS (SELECT 1 FROM AuditEvent ae WHERE l.id = ae.licenceId AND ae.detail LIKE '%APPROVED%')) 
+         OR (l.versionOfId IS NOT NULL) 
+        )
     """,
```

Comment:
This query finds in progress licences that have been previously approved. When versioning is enabled in progress licence versions won't have an approved audit event as this will be linked to the previous licence version but they will have versionOfId populated.